### DISCUSSION TITLE
Are GitHub Icons Open Source

### DISCUSSION THREAD

**Discussion by jessiejs** [community/community#41579] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Are the icons at https://ghicons.github.com/ publically available like Octicons, and the rest of the github design system?

**Comment by JustOptimize** [comment]
There is a repo link in that page but it returns a 404 (I guess it has been set to private)
Link: https://github.com/github/icons

**Comment by jessiejs** [comment]
I know that, but are people outside of github allowed to use these?

**Reply by JustOptimize** [reply]
Since the repo is no longer available, you are in a gray area.
It should be allowed, but since you can't see the license you can't be 100% sure.
The site there has no trademark, no footer or terms and has 1250 empty lines in the html source 🤷🏻
